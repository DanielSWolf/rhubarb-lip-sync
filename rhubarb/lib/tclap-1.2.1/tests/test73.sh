#!/bin/sh

# success  tests whether * in UnlabeledValueArg passes 
../examples/test2 -i 1 -s asdf fff*fff > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test73.out; then
	exit 0
else
	exit 1
fi

