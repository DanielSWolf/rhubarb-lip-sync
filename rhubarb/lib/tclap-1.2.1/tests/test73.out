for float we got : 3.7
for int we got : 1
for string we got : asdf
for ulabeled we got : fff*fff
for bool A we got : 0
for bool B we got : 0
for bool C we got : 0
